### ISSUE TITLE
[onert] An error by using SubTensor as input of Pack operation on acl backend

### ISSUE THREAD

**Issue by ragmani** [Samsung/ONE#4244]
![Subtensor error](https://user-images.githubusercontent.com/18050611/93043957-54664980-f68e-11ea-9921-cc066d796020.PNG)

```bash
$ USE_NNAPI=1 BACKENDS="acl_cl" ./Product/armv7l-linux.debug/out/bin/nnapi_test temp.tflite 
[NNAPI TEST] Run T/F Lite Interpreter without NNAPI
[NNAPI TEST #1] Run T/F Lite Interpreter with NNAPI
...
[ALLOC] base pointer: 0x30b870
[acl_cl_KernelGenerator_Concat] Concat eliminated
[EXCEPTION] in extend_parent_shape src/core/SubTensorInfo.cpp:45: (coords.x() != 0) || (coords.y() != 0)
[NNAPI::Compilation] finish: Fail to compile
Aborting since NNAPI returned failure nnapi_delegate.cc:1098
[onert_backend_create] 'acl_cl' unloaded
```

**Comment by ragmani** [comment]
Done

**Comment by ragmani** [comment]
#4266 will be fix the error caused by #4266
